ffi/tests/test_expression_visitor/run_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Can you make an issue for unifying our c test code (i.e. this script could be probably shared between the two tests).

Could be a good first issue probably

Reply:
Done #427